File: HolaApp/app/src/main/java/com/example/holaapp/MainActivity.java
package com.example.holaapp;

import android.os.Bundle;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;
//importaciones de las vistas
import android.widget.Button;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        editText valA = findViewById(R.id.valA);
        editText valB = findViewById(R.id.valB);
        Button btnMul = findViewById(R.id.btnMul);
        TextView tvRes = findViewById(R.id.tvres);

        btnMul.setOnClickListener(new View.OnClickListener());
        @Override
        public void onClick(View view){
            String a = valA.getText().toString().trim();

        }
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });
    }
}

File: HolaAppkt/app/src/main/java/com/example/holaappkt/MainActivity.kt
package com.example.holaappkt

import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val valA = findViewById<EditText>(R.id.valA)
        val valB = findViewById<EditText>(R.id.valB)
        val btnMul = findViewById<Button>(R.id.btnMul)
        val tvRes = findViewById<TextView>(R.id.tvRes)

        btnMul.setOnClickListener{
            val a = valA.text.toString().trim()
            val b = valB.text.toString().trim()

            if(a.isNotEmpty() && b.isNotEmpty()){
                val result = a.toDouble() * b.toDouble()
                tvRes.text = "Resultado: $result"
            }
        }



//        enableEdgeToEdge()
//        setContentView(R.layout.activity_main)
//        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
//            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
//            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom) //           insets
//        }
    }
}